src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  function getSum(firstNum) {
+    counter++;
+
+    return (secondNum) => {
+      if (counter > 3 && counter % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return firstNum + secondNum;
```

Comment:
You can use the ternary operator to write this condition, but this is optional ;)